sdk/communication/azure-communication-rooms/azure/communication/rooms/_generated/aio/_client.py

```diff
@@ -34,15 +35,34 @@ def __init__(  # pylint: disable=missing-client-constructor-parameter-credential
     ) -> None:
         _endpoint = "{endpoint}"
         self._config = AzureCommunicationRoomsServiceConfiguration(endpoint=endpoint, **kwargs)
-        self._client: AsyncPipelineClient = AsyncPipelineClient(base_url=_endpoint, config=self._config, **kwargs)
+        _policies = kwargs.pop("policies", None)
+        if _policies is None:
```

Comment:
I believe this is a new generated codes. I do not think it will affect the changes but to ensure, please run all tests and sample scenarios.